.github/workflows/build_config_server.yaml

```diff
@@ -31,21 +31,14 @@ jobs:
     - uses: actions/checkout@v4
 
     - name: Build Image
-      run: ./build.ps1 -Name ${{ env.IMAGE_NAME }} -Registry ${{ env.REGISTRY }}
+      run: ./build.ps1 -Name ${{ env.IMAGE_NAME }} -Registry ${{ env.REGISTRY }} -DirectoryName "config-server"
 
-    - name: Login to private container registry
-      if: ${{ github.event_name == 'pull_request' }}
+    - name: Login to container registry
       uses: docker/login-action@v3
       with:
         registry: "${{ vars.DOCKER_REGISTRY }}"
         username: "${{ secrets.DOCKER_USERNAME }}"
         password: "${{ secrets.DOCKER_PASSWORD }}"
-    - name: Login to Docker Hub
```

Comment:
`build_uaa_server.yaml` wasn't updated accordingly, is that intentional?

Reply:
only in the sense I didn't want to do _anything_ with that one right now since it's much more involved to test that nothing broke when updating the dependencies